lldb/source/Symbol/SymbolFileOnDemand.cpp

```diff
@@ -555,6 +555,18 @@ StatsDuration::Duration SymbolFileOnDemand::GetDebugInfoIndexTime() {
   return m_sym_file_impl->GetDebugInfoIndexTime();
 }
 
+void SymbolFileOnDemand::ResetDebugInfoParseTime() {
+  LLDB_LOG(GetLog(), "[{0}] {1} is not skipped", GetSymbolFileName(),
+           __FUNCTION__);
+  return m_sym_file_impl->ResetDebugInfoParseTime();
+}
+
+void SymbolFileOnDemand::ResetDebugInfoIndexTime() {
+  LLDB_LOG(GetLog(), "[{0}] {1} is not skipped", GetSymbolFileName(),
+           __FUNCTION__);
+  return m_sym_file_impl->ResetDebugInfoIndexTime();
+}
+
```

Comment:
Switch to a single function `ResetStatistics` and fix the log text "is not skipped" seems wrong?

Reply:
"is not skipped" is correct -- it means this API call `ResetStatistics` is not skipped by SymbolFileOnDemand class and pass through to the underlying real symbol file.